sdk/vc-data/src/data/accountOwnership/v1.ts

```diff
@@ -0,0 +1,106 @@
+import { CreateThing, createContextEntry } from '../util'
+
+export type AccountOwnershipV1 = CreateThing<
+  'AccountOwnership',
+  {
+    accountName: 'github' | 'twitter',
+    accountType: 'github' | 'twitter',
+    metaData: AccountOwnershipGithubV1
+  }
+  >
+
+export type AccountOwnershipGithubV1 = CreateThing<
+  'AccountOwnershipGithub',
+  {
+    id?: number
+    login?: string
+    url?: string
+    name?: string
+    email?: string
+    followers?: number
+    following?: number
+    twitterUsername?: string
```

Comment:
Do we need `twitterUsername` in `AccountOwnershipGithub`, or is it a mistype?

Reply:
This field from the user's GitHub account might be useful I think. And we can combine it with verification of Twitter account ownership in future